src/main/java/core/basesyntax/dao/impl/CommentDaoImpl.java

```diff
@@ -12,21 +16,64 @@ public CommentDaoImpl(SessionFactory sessionFactory) {
 
     @Override
     public Comment create(Comment entity) {
```

Comment:
```suggestion
    public Comment create(Comment comment) {
```
better to rename all parameter coz of clear understanding, what parameter you will work with in your method